source/GameData.h

```diff
@@ -162,6 +163,7 @@ class GameData {
 	static const std::string &Rating(const std::string &type, int level);
 	// Collections for ship, bay type, outfit, and other categories.
 	static const CategoryList &GetCategory(const CategoryType type);
+	static const BayType *GetBayType(const std::string &type);
```

Comment:
this is really not in line with the other getters of gamedata,
as this is basically just another data object, it should have the same type of getter as everything else.